Snakefile

```diff
@@ -2,44 +2,25 @@
 #
 # SPDX-License-Identifier: MIT
 
-from os.path import normpath, exists
-from shutil import copyfile, move, rmtree
```

Comment:
needs to be imported in _helpers.py

Reply:
it was the `move` function which is needed in some smk files (and the rmtree is used as well). Reverted this